packages/components/src/core/NavigationHeader/NavigationHeader.types.ts

```diff
@@ -0,0 +1,28 @@
+import { ButtonProps, LinkProps } from "@mui/material";
+import { ReactNode, ElementType } from "react";
+import { SdsProps, SdsIconButtonProps } from "../Button";
+import { InputSearchProps } from "../InputSearch";
+import { SdsTagColorType } from "../Tag";
+import { NavigationHeaderPrimaryNavItem } from "./components/NavigationHeaderPrimaryNav";
+import { NavigationHeaderSecondaryNavItem } from "./components/NavigationHeaderSecondaryNav";
+
+export interface NavigationHeaderProps<T extends string = string> {
```

Comment:
Sanity check: Do we need to export this type from the component index file lol? I forgot 😂 Thank you!

Reply:
Oh right! 😯 thanks for catching that! We definitely need to export the types. 🥳 🤩